docker-compose.yml

```diff
@@ -28,6 +28,7 @@ services:
     command: >
       bash -c "wait-for-it -t 30 db:5432 &&
                rm -f tmp/pids/server.pid &&
+               yarn install &&
```

Comment:
this is in the dockerfile already. Is this really needed? Maybe we should remove it from the dockerfile then?

Reply:
I'm not sure why "yarn install" has to be in docker-compose.yml as well, but it really is needed.  Without the change in this  pull request, I get the error message instead of the expected app display.  The change I made in this pull request fixes the problem.

Reply:
someone complaining about this error in slack. I am going to merge this PR now :+1: Thanks!